Stencil/Filters.swift

```diff
@@ -8,7 +8,7 @@ func toString(value: Any?) -> String? {
   return nil
 }
 
-func capitalise(value: Any?) -> Any? {
+func capitalise(value: Any?, args: [String]?) -> Any? {
```

Comment:
Since filters with no arguments will be more common, I think we should have a `registerFilter` function inside the `Namespace` which will accept filters without arguments so this change would be unnecessary. It should wrap the given filter in a closure which accepts the arguments and throws if arguments have been presented.